examples/js/exporters/GLTFExporter.js

```diff
@@ -1777,7 +1777,7 @@ THREE.GLTFExporter.prototype = {
 
 				}
 
-				if ( ! equalArray( object.matrix.elements, [ 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1 ] ) ) {
+				if ( ! equalArray( object.matrix.elements, new THREE.Matrix4().toArray() ) ) {
```

Comment:
In this case why not:
```js
if ( object.matrix.equals( THREE.Matrix4() ) === false ) ) {
```

Reply:
Or maybe just just this somewhere in `GLTFExporter`:

```js
const identityArray = [ 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1 ];

function isIdentityMatrix( matrix ) {

    return equalArray( matrix.elements, identityArray );

}
```